app/views/admin/api_docs/base/swagger.html.erb

```diff
@@ -37,18 +37,80 @@
       }
     }
 
+    var AUTH_METHOD_USER_KEY = 1;
```

Comment:
`const`?

Reply:
I am very confused about this `var` vs `const` or `let` in our case. We removed all ES6/7 stuff for capybara, right? But since we don't use Jenkins anymore, we can come back to add ES6/7 stuff, right? @josemigallas @damianpm @didierofrivia @guicassolato

Reply:
This file uses vanilla JS (no fancy modern syntax), we let ES6 features for `.jsx` files since they are transpiled by babel.